content/scout/explore/exceptions.md

```diff
@@ -21,272 +20,20 @@ In Docker Scout, exceptions are automatically factored into the results.
-registry. You don't need to build or push the image again. Additionally, having
-the exceptions attached to the image as attestations means consumers can
-inspect the exceptions for an image, directly from the registry.
-
-To attach an attestation to an image:
-
-1. Build the image and push it to a registry.
-
-   ```console
-   $ docker build --provenance=true --sbom=true --tag <IMAGE> --push .
-   ```
-
-2. Attach the exception to the image as an attestation.
-
-   ```console
-   $ docker scout attestation add \
-     --file <cve-id>.vex.json \
-     --predicate-type https://openvex.dev/ns/v0.2.0 \
-     <IMAGE>
-   ```
-
-   The options for this command are:
-
-   - `--file`: the location and filename of the VEX document
-   - `--predicate-type`: the in-toto `predicateType` for OpenVEX
-
-### Image filesystem
-
-Embedding exceptions directly on the image filesystem is a good option if you
-know the exceptions ahead of time, before you build the image. And it's easy;
-just `COPY` the VEX document to the image in your Dockerfile.
-
-The downside with this approach is that you can't change or update the
-exception later. Image layers are immutable, so anything you put in the image's
-filesystem is there forever. Attaching the document as an
-[attestation](#attestation) provides better flexibility.
-
-> **Note**
->
-> VEX documents embedded in the image filesystem are not considered for images
-> that have attestations. If your image has **any** attestations, Docker Scout
-> will only look for exceptions in the attestations, and not in the image
-> filesystem.
->
-> If you want to use the VEX document embedded in the image filesystem, you
-> must remove the attestation from the image. Note that provenance attestations
-> may be added automatically for images. To ensure that no attestations are
-> added to the image, you can explicitly disable both SBOM and provenance
-> attestations using the `--provenance=false` and `--sbom=false` flags when
-> building the image.
-
-To embed a VEX document on the image filesystem, `COPY` the file into the image
-as part of the image build. The following example shows how to copy all VEX
-documents under `.vex/` in the build context, to `/var/lib/db` in the image.
-
-```dockerfile
-# syntax=docker/dockerfile:1
-
-FROM alpine
-COPY .vex/* /var/lib/db/
-```
-
-The filename of the VEX document must match the `*.vex.json` glob pattern.
-It doesn't matter where on the image's filesystem you store the file.
-
-Note that the copied files must be part of the filesystem of the final image,
-For multi-stage builds, the documents must persist in the final stage.
+The recommended way to create exceptions is to use Docker Scout Dashboard or
+Docker Desktop. The GUI provides a user-friendly interface for creating
+exceptions, and it's easy to manage exceptions for multiple images. It also
+lets you create exceptions for multiple images, or your entire organization,
```

Comment:
Not a strong objection, but the "manage exceptions for multiple images" and "create exceptions for multiple images" feel a bit repetitive in these two consecutive sentences.